downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/BiologicalProcessAnnotationBuilder.java

```diff
@@ -118,25 +117,25 @@ private static void processProteins(Set<GKInstance> proteinInstances, GKInstance
     /**
      * Before creating GOA line for BP annotations, the reaction in question needs to be checked for the existance of a 'goBiologicalProcess' attribute. If there is none
      * than the instance's 'hasEvent' referrals are checked for any.
-     * @param referenceEntityInst -- ReferenceEntity instance from protein instance.
-     * @param reactionInst
-     * @param taxonIdentifier -- CrossReference ID of protein's species.
-     * @throws Exception
+     * @param referenceEntityInst -- GKInstance, ReferenceEntity instance from protein instance.
+     * @param reactionInst -- GKInstance, parent reaction instance.
+     * @param taxonIdentifier -- String, CrossReference ID of protein's species.
+     * @throws Exception -- MySQLAdaptor exception.
      */
     private static void getGOBiologicalProcessLine(GKInstance referenceEntityInst, GKInstance reactionInst, String taxonIdentifier) throws Exception {
         for (Map<String, String> biologicalProcessAccession : getGOBiologicalProcessAccessions(reactionInst, 0)) {
-            String goaLine = GOAGeneratorUtilities.generateGOALine(referenceEntityInst, BIOLOGICAL_PROCESS_LETTER, biologicalProcessAccession.get("accession"), biologicalProcessAccession.get("event"), "TAS", taxonIdentifier);
+            String goaLine = GOAGeneratorUtilities.generateGOALine(referenceEntityInst, GOAGeneratorConstants.BIOLOGICAL_PROCESS_LETTER, biologicalProcessAccession.get(GOAGeneratorConstants.ACCESSION_STRING), biologicalProcessAccession.get(GOAGeneratorConstants.EVENT_STRING),  GOAGeneratorConstants.TRACEABLE_AUTHOR_STATEMENT_CODE, taxonIdentifier);
```

Comment:
Could the constants from `GOAGeneratorConstants` be statically imported so the class name isn't needed as a prefix each time?

Reply:
Done